### ISSUE TITLE
Are Vulkan and OpenGL applications supported?

### ISSUE THREAD

**Issue by vinjn** [GameTechDev/PresentMon#48]

**Comment by jdm3** [comment]
Yes, they should capture fine on Windows so let us know if you see otherwise.  I'll update the readme.

**Comment by vinjn** [comment]
On my Win10, neither Vulkan nor OpenGL app can be captured.
Do you need ETL to repro?

**Comment by jdm3** [comment]
Yes, if you can provide one that's the best way to debug.
